crash_similarity.py

```diff
@@ -212,9 +213,9 @@ def signature_similarity(model, paths, signature1, signature2):
         for doc2 in traces2:
             words2 = [word for word in preprocess(doc2) if word in model]
 
-            if words1 == words2 or frozenset([frozenset(words1), frozenset(words2)]) in already_processed:
+            if words1 == words2 or frozenset([tuple(words1), tuple(words2)]) in already_processed:
```

Comment:
if they are frozen set the order in which the words appear does not matter,
Ex
function1, function2 function3 == function2, function3, functioin1